tbfparser/src/lib.rs

```diff
@@ -682,7 +679,7 @@ impl TbfHeader {
         let current_length = header_buf.get_ref().len();
         util::do_pad(
             &mut header_buf,
-            amount_alignment_needed(current_length as u32, 4) as usize,
+            util::amount_alignment_needed(current_length as u32, 4) as usize,
```

Comment:
```suggestion
            amount_alignment_needed(current_length as u32, 4) as usize,
```
Add `use util::amount_alignment_needed` at the top of the file.

Reply:
Done